carma_wm/test/CARMAWorldModelTest.cpp

```diff
@@ -39,6 +39,9 @@ namespace carma_wm
 void createTestingWorld(carma_wm::CARMAWorldModel& cmw, std::vector<lanelet::Lanelet>& llts,
                                           lanelet::LaneletMapPtr& map,  std::vector<cav_msgs::ExternalObject>& obstacles)
 {
+  //TODO: cmw is not used; if there are no plans to use it soon, remove it from param list
+  //TODO: map is overwritten before it is used, so it has no value as an input param; is it intended to be an output?
```

Comment:
I think all 4 parameters are output parameters here which is why all of them are non-const references. 
cmw missing seems like an oversight. If its not used it can be removed.

Reply:
cmw removed.